android/src/main/java/com/amplitude/android/utilities/DefaultEventUtils.kt

```diff
@@ -0,0 +1,157 @@
+package com.amplitude.android.utilities
+
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.PackageInfo
+import android.content.pm.PackageManager
+import android.net.ParseException
+import android.net.Uri
+import android.os.Build
+import com.amplitude.android.Amplitude
+import com.amplitude.core.Storage
+import kotlinx.coroutines.launch
+
+class DefaultEventUtils(private val amplitude: Amplitude) {
+    object EventTypes {
+        const val APPLICATION_INSTALLED = "[Amplitude] Application Installed"
+        const val APPLICATION_UPDATED = "[Amplitude] Application Updated"
+        const val APPLICATION_OPENED = "[Amplitude] Application Opened"
+        const val APPLICATION_BACKGROUNDED = "[Amplitude] Application Backgrounded"
```

Comment:
Application Closed?

Reply:
Good question, my understanding is application closed event is either not supported or flaky (https://stackoverflow.com/questions/33325035/android-application-kill-event-capture), probably it is why not commonly adopted (see [doc](https://amplitude.atlassian.net/wiki/spaces/GROWTH/pages/2103935062/Default+Event+Tracking+for+Mobile+SDKs#Which-default-events%3F)). Application backgrounded should have similar indication as it relies on `onActivityStopped` hook.